web/less/icons.less

```diff
@@ -155,12 +161,12 @@
     height: 27px;
     margin-left: 0;
   }
-   @media @iphoneAll @Landscape {
-      background-image: url("/images/vonage-v-logo.svg");
-      width: 32px;
-      height: 27px;
-      margin-left: 0;
-    }
+  @media @iphoneAll @Landscape {
```

Comment:
@Raulkg I am thinking that we should delete the `@iphoneAll` variable and be making these changes for `@smartphones` (or for `@smartphonesLandscape` or `@smartphonesPortrait`. Wdyt?

Reply:
the smartphone landscape doesn't work for specific landscape feature as it is combining several other features like pixel ratio and doesn't give power over control for landscape orientation on various devices